src/app/components/elements/TagList.jsx

```diff
@@ -14,6 +14,7 @@ export default ({ post, horizontal, single }) => {
 
     const json = post.json_metadata;
     let tags = [];
+    tags = tags.filter(tag => typeof tag === 'string');
```

Comment:
`tags` is empty at this point, so this filter is not necessary here

Reply:
yes, this line is kind of pointless.